content/influxdb/cloud-serverless/reference/client-libraries/v3/java.md

```diff
@@ -21,3 +21,81 @@ to write and query data stored in an {{% product-name %}} bucket.
 The documentation for this client library is available on GitHub.
 
 <a href="https://github.com/InfluxCommunity/influxdb3-java" target="_blank" class="btn github">InfluxDB v3 Java client library</a>
+
+## Installation
+
+### Maven
+```xml
+<dependency>
+  <groupId>com.influxdb</groupId>
+  <artifactId>influxdb3-java</artifactId>
+  <version>RELEASE</version>
+</dependency>
+```
+
+### Gradle
+```groovy
+implementation group: 'com.influxdb', name: 'influxdb3-java', version: 'latest.release'
+```
+
+## Sample Code
+
+```java
+public class HelloInfluxDB {
+  private static final String HOST_URL = "https://eu-central-1-1.aws.cloud2.influxdata.com"; // URL information is present in the cloud portal
```

Comment:
The docs do one of the following to automatically populate cloud URLs:

- If the user is referred to the docs from their InfluxDB Cloud dashboard, we detect the referrer and automatically set their URL.
- The user can manually select their region and we'll automatically populate the URL.

```suggestion
  private static final String HOST_URL = "https://{{< influxdb/host >}}";
```